app/lib/tufts/workflow_status.rb

```diff
@@ -0,0 +1,33 @@
+module Tufts
+  module WorkflowStatus
+    def self.status(id)
```

Comment:
Could we put this in the presenter, which already has access to the solr document? I'm concerned about the impact of calling solr queries from the view, especially for search result badges.

Reply:
I feel like I need to use this in places where I don't have access to a presenter, but I might be wrong. I'll double-check

Reply:
The catalog view is using the partial in `app/views/catalog/_batch_select.html.erb` now

Reply:
re: `app/views/catalog/_batch_select.html.erb`

Do we want to change this? It seems odd to have the draft/workflow status in the batch select partial.